.github/workflows/suggest_release.yml

```
@@ -0,0 +1,39 @@
+name: Suggest auto-releasing
+on:
+  pull_request:
+    branches:
+      - main
+    types:
+      - opened
+    paths:
+      - 'attractions/package.json'
+
+jobs:
+  tag-issue:
+    name: Tag the PR with [new release] if needed
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v2
+        with:
+          persist-credentials: false
+
+      - name: Post the URL on the PR 🖊
+        uses: actions/github-script@v3
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const package = JSON.parse(require('fs').readFileSync('attractions/package.json'));
```

Same as in other PR:
 ```suggestion
            const package = require(`${process.env.GITHUB_WORKSPACE}/attractions/package.json`);
```